proc/proc_linux.go

```diff
@@ -338,12 +339,32 @@ func (dbp *Process) loadProcessInformation(wg *sync.WaitGroup) {
 	defer wg.Done()
 
 	comm, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/comm", dbp.Pid))
-	if err != nil {
+	if(err == nil) {
+		// removes newline character
+		comm = comm[:len(comm)-1]
+	}
+	for (comm == nil || len(comm) <= 0) {
```

Comment:
Right, logically it's if. I meant to use break statement to leave current block when error occur.